Okta.Xamarin/Okta.Xamarin.iOS/OktaViewController.cs

```diff
@@ -59,13 +59,13 @@ public override void ViewDidLoad()
 				$"Scope: {string.Join(", ", config.Scopes)}\n" +
 				$"ClockSkew: {config.ClockSkew.ToString()}\n";*/
 
-			client = new OidcClient(this, config);
+			client = new iOsOidcClient(this, config);
 		}
 
-		public async Task<OktaStateManager> SignIn(IOktaConfig oktaConfig = default)
+		public async Task<IOktaStateManager> SignIn(IOktaConfig oktaConfig = default)
 		{
-			oktaConfig = oktaConfig ?? OktaConfig.LoadFromPList("OktaConfig.plist");
-			OidcClient oidcClient = new OidcClient(this, oktaConfig);
+			oktaConfig = oktaConfig ?? iOsOktaConfig.LoadFromPList("OktaConfig.plist");
+			OidcClient oidcClient = new iOsOidcClient(this, oktaConfig);
 			return await oidcClient.SignInWithBrowserAsync();
```

Comment:
Just thinking. `await` after `return` may not be needed because caller will await this function anyway, and since this is the only awaited call in this function, the function itself can be made synchronous.

Reply:
Making things synchronous in Xamarin (iOS especially) is potentially detrimental to the UI, causing it to visibly "lock up" at times.  I think it's best to try to be async by default at that level of an application.